src/components/AssetManagementPanel/AssetManagementForm/FormVariants/EditableAssetTitle/EditableAssetTitle.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { render, fireEvent, act } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { EditableAssetTitle } from "./EditableAssetTitle";
+
+it("should display both the role and the value when not in editable format", () => {
+  const container = render(
+    <EditableAssetTitle
+      role="Beneficiary"
+      value="0xE94E4f16ad40ADc90C29Dc85b42F1213E034947C"
+      isEditable={false}
+      onSetNewValue={() => {}}
```

Comment:
Yep.. on second look, let's have it nullable

Reply:
updated it with: onSetValue? : .... | null
updated value with value? string | null

Reply:
Can reflect this in the test as well